Module_1/Hello_Microverse/.github/workflows/index.html

```diff
@@ -7,6 +7,7 @@
     <title>Document</title>
 </head>
 <body>
+<!-- This is a Heading     -->
     <h1>HELLO Microverse</h1>
```

Comment:
- Please change this to `Hello Microverse!` as required
- Please set up your linters. as specified [here](https://github.com/microverseinc/linters-config/tree/master/html-css)
- Please add a professional readme. use this [template](https://github.com/microverseinc/readme-template)
- Please add your  `.gitignore` file